pkg/controller/networkpolicy/store/addressgroup.go

```diff
@@ -162,6 +162,22 @@ func NewAddressGroupStore() storage.Interface {
 			// ag.Selector.Namespace == "" means it's a cluster scoped group, we index it as it is.
 			return []string{ag.Selector.Namespace}, nil
 		},
+		PodIndex: func(obj interface{}) ([]string, error) {
+			ag, ok := obj.(*types.AddressGroup)
+			if !ok {
+				return []string{}, nil
+			}
+			keys := make([]string, 0)
```

Comment:
if you are going to create a slice of length 0, just use `make([]string)`

Reply:
It gives a compilation error if I don't specify length

Reply:
that's right my bad